contrib/babelfishpg_common/src/encoding/mb/conv.c

```diff
@@ -271,6 +277,12 @@ TsqlUtfToLocal(const unsigned char *utf, int len,
 		}
 		iutf = (b1 << 24 | b2 << 16 | b3 << 8 | b4);
 
+		if (!pg_utf8_islegal(cur, l))
```

Comment:
Just curious, how are we making sure that our test cases covers case which hits "break" previously?